### PR TITLE
Add --useDevPrivilegeKeys option to enable usage of dev keys for testing

### PR DESCRIPTION
**Pull request by mrosseel**
* added parser option in both BisqAppMain and BisqExecutable
* made USE_DEV_PRIVILEGE_KEYS non-final

### CODE REVIEW

File: gui/src/main/java/io/bisq/gui/app/BisqAppMain.java [hunk 1]
```diff
@@ -48,6 +49,12 @@ public static void main(String[] args) throws Exception {
                 .withRequiredArg();
         parser.accepts(AppOptionKeys.APP_NAME_KEY, description("Application name", DEFAULT_APP_NAME))
                 .withRequiredArg();
+        parser.accepts(AppOptionKeys.USE_DEV_PRIVILEGE_KEYS,
+                description("Dev mode, all the privileged features which require a private key to enable it " +
+                        "are overridden by a dev key pair. Features: Arbitration registration" +
+                        " (alt+R at account), Alert/Update (alt+m), private message to a peer " +
+                        "(click user icon and alt+r), filter/block offers by various data like" +
+                        " offer ID (cmd + f)", false));
```
Comment (ManfredKarrer) [inline gui/src/main/java/io/bisq/gui/app/BisqAppMain.java:57]: This is not needed here.

File: gui/src/main/java/io/bisq/gui/app/BisqAppMain.java [hunk 2]
```diff
@@ -68,6 +75,10 @@ public static void main(String[] args) throws Exception {
         // In order to work around a bug in JavaFX 8u25 and below, you must include the following code as the first line of your realMain method:
         Thread.currentThread().setContextClassLoader(BisqAppMain.class.getClassLoader());
 
+        if(options.has(AppOptionKeys.USE_DEV_PRIVILEGE_KEYS)) {
+            DevEnv.USE_DEV_PRIVILEGE_KEYS = true;
+        }
+
```
Comment (ManfredKarrer) [inline gui/src/main/java/io/bisq/gui/app/BisqAppMain.java:81]: This is not needed here.

But all the other places where the option keys are applied are missing.

**Comment by ManfredKarrer** [comment]
Not complete for merge. See comments. Just follow other option keys to see where changes have to be applied.

**Comment by mrosseel** [comment]
Ok, will do.

**Pull request closed without merging**